backend/pkg/metrics/README.md

```diff
@@ -0,0 +1,32 @@
+# Prometheus
+
+In order to expose metrics from KubeClarity to Prometheus, you need to enable
+the endpoint by setting the environment variable `PROMETHEUS_REFRESH_INTERVAL_SECONDS`
+to value larger than zero. There is no particular need to scrape these metrics very
+often, so once every 300 seconds is fine.
+
+You can set this value in the `values.yaml` file.
```

Comment:
Need to mention that there is a need to also enable the Prometheus section in `values.yaml`, right?